lib/bgs/base.rb

```diff
@@ -167,7 +167,7 @@ def request(method, message = nil, identifier = nil)
 
         file_path = BGS.configuration.mock_response_location
         file_path += "/#{@service_name.underscore}/#{method}/#{identifier}.json"
-        OpenStruct.new(body: JSON.parse(File.read(file_path)).with_indifferent_access)
+        Struct.new(body: JSON.parse(File.read(file_path)).with_indifferent_access)
```

Comment:
this was from a linting failure I think related to #68. Seem fine to update without issue.